src/navigators/createMaterialTopTabNavigator.js

```diff
@@ -113,18 +133,109 @@ class TabView extends React.PureComponent<Props> {
 
   _renderPanPager = props => <TabViewPagerPan {...props} />;
 
-  _renderScene = ({ route, focused }) => {
-    const { renderScene, animationEnabled, swipeEnabled } = this.props;
+  componentWillMount() {
```

Comment:
`componentWillMount` is deprecated. We should use `componentDidMount` to do side-effects such as adding a listener.